src/common/dune

```diff
@@ -0,0 +1,5 @@
+(library
+ (name             AltErgoCommon)
```

Comment:
I would advice to always use lowercase for libraries (and also to use only `_` and not `-` but this isn't relevant for that one).